tests/test_feature.rb

```diff
@@ -97,7 +85,13 @@ def test_nv_overlay
       assert_raises(Cisco::UnsupportedError) { Feature.nv_overlay_enable }
       return
     end
+    # Test if VxLAN can be configured
+    vxlan_linecard?
+    vdc_current = node.product_id[/N7/] ? vdc_lc_state : nil
+    vdc_lc_state('f3') if vdc_current
     feature('nv_overlay')
+    # Return testbed to pre-clean state
+    vdc_lc_state(vdc_current) if vdc_current
```

Comment:
Okay, I'm mostly okay with these changes. Just one thing I don't like. Deepak and I were discussing earlier in the week that returning the testbed to the previous state was problematic, particularly when a test fails midway through, then you start a new test, in which case the previous state may not be what you want. So we decided we should just set it to the state we want and leave it in that state.

In other words, change the above code to just this:

```
+    # Test if VxLAN can be configured
+    vxlan_linecard?
-    vdc_current = node.product_id[/N7/] ? vdc_lc_state : nil
-    vdc_lc_state('f3') if vdc_current
+    vdc_lc_state('f3') if node.product_id[/N7/]
      feature('nv_overlay')
-    # Return testbed to pre-clean state
-    vdc_lc_state(vdc_current) if vdc_current
```